js/login.js

```diff
@@ -0,0 +1,111 @@
+import { validateEmail, isPasswordValid, isEmpty } from "/js/common/formValidator.js";
+
+const emailInput = document.getElementById("inputEmail");
+const pwdInput = document.getElementById("inputPwd");
+
+const emailValidError = document.getElementById("emailValidError");
+const pwdValidError = document.getElementById("pwdValidError");
+
+const loginBtn = document.getElementById("loginBtn");
+
+const EMAIL_EMPTY_ERROR_MSG = "이메일을 입력해주세요.";
+const EMAIL_VALID_ERROR_MSG = "잘못된 이메일 형식입니다.";
+const PWD_EMPTY_ERROR_MSG = "비밀번호를 입력해주세요.";
+const PWD_LENGTH_ERROR_MSG = "비밀번호를 8자 이상 입력해주세요.";
+
+// 에러 메세지 표시
+function showError(inputEl, errorEl, message) {
+  inputEl.classList.add('error');
+  inputEl.classList.remove("success");
+  errorEl.textContent = message;
+};
+
+// 에러 메세지 제거
+function clearError(inputEl, errorEl) {
+  inputEl.classList.remove('error');
+  inputEl.classList.add('success');
+  errorEl.textContent = '';
+};
+
+/**
+ * 입력값들의 유효성을 검사한 후,
+ * 모든 항목이 유효할 경우 회원가입 버튼이 활성화
+ * 그렇지 않으면 비활성화
+ */
+function toggleButtonState() {
+  const isEmailValid = !isEmpty(emailInput.value) && validateEmail(emailInput.value);
+  const isPwdValid = !isEmpty(pwdInput.value) && isPasswordValid(pwdInput.value);
+
+  const isFormValid = isEmailValid && isPwdValid;
+
+  if (isFormValid) {
+    loginBtn.setAttribute("aria-disabled", "false");
+    loginBtn.classList.remove("disabled");
+  } else {
+    loginBtn.setAttribute("aria-disabled", "true");
+    loginBtn.classList.add("disabled");
+  }
+}
+
+// 초기 상태에서 회원가입 버튼 비활성화 처리
+document.addEventListener("DOMContentLoaded", () => {
+  loginBtn.setAttribute("aria-disabled", "true");
+  loginBtn.classList.add("disabled");
+  setupPasswordToggle();
+});
+
+// 이메일 유효성 검사
+emailInput.addEventListener("input", () => {
+  if (isEmpty(emailInput.value)) {
+    showError(emailInput, emailValidError, EMAIL_EMPTY_ERROR_MSG);
+    return;
+  } else if (!validateEmail(emailInput.value)) {
+    showError(emailInput, emailValidError, EMAIL_VALID_ERROR_MSG);
+    return;
+  } else {
+    clearError(emailInput, emailValidError);
+  }
+  toggleButtonState();
+});
```

Comment:
❗️ 수정요청
중간 중간에 return문을 사용하고 있으셔서 else if문을 쓰실 필요가 없어 보입니다~

```suggestion
emailInput.addEventListener("input", () => {
  if (isEmpty(emailInput.value)) {
    showError(emailInput, emailValidError, EMAIL_EMPTY_ERROR_MSG);
    return; // 값이 없을 경우 에러메시지를 보여주고 로직 종료
  }
  if (!validateEmail(emailInput.value)) {
    showError(emailInput, emailValidError, EMAIL_VALID_ERROR_MSG);
    return;
  } 
  clearError(emailInput, emailValidError);
  toggleButtonState();
});
```